.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1816,6 +1968,94 @@ stages:
             command: "CheckBuildLogsForErrors"
             apiKey: $(DD_LOGGER_DD_API_KEY)
 
+- stage: integration_tests_arm64_debugger
```

Comment:
Even though we're not adding these stages to the coverage stage, we should add them to the `trace_pipeline` stage.

Reply:
Oh, right. Fixed here 936021ce2e8f838ee0f712209a1f3330bbd0f408